install_rock.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-if [ "$(uname -o)" == "Msys" ]; then
-	tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*zip\>')"
-	wget https://github.com/cogneco/rock/releases/$tmp
-	echo ${tmp##*/}
-	7z e ${tmp##*/}
-else
```

Comment:
We should keep support for Windows as long as the impact on our speed forward is low. We might need it soon again.

Support for running without GPU is probably even more interesting, long term.